scss/components/_activity-panel.scss

```diff
@@ -56,6 +56,22 @@
       margin-top: 0;
     }
   }
+
+  &.civicase__activity__status-draft .help {
```

Comment:
The `.help` class is already hidden when inside the `civicase__activity-panel__core_container`. So the code to show it should be inside the same.

so in https://github.com/compucorp/uk.co.compucorp.civicase/blob/73e0686f5f95b89795fe9fe145727a63faca3bc7/scss/components/_activity-panel.scss#L118

you can do


```
&:not(.civicase__activity-panel__core_container--draft) {
  .help {
     display: none;
  }
}
```

that way no need to define the same styles again.